docs/faq.md

```diff
@@ -127,7 +127,7 @@ Semgrep CI makes network requests in accordance with the data storage mentioned
 # Configuration
 
 #### How do I configure Semgrep for different projects?
-Semgrep Community and Team editions provide centralized policy management. See [Managing CI policy](managing-policy.md) for more details.
+Semgrep App’s Community and Team tiers provide centralized policy management. See [Managing CI policy](managing-policy.md) for more details.
```

Comment:
Is it necessary to mention Community and Team here or could this be shortened to "Semgrep App provides centralized"? We could go further and reference the specific feature: Semgrep App's Dashboard

Reply:
I’m fine with that change. I was looking to find _some_ place to introduce the tiers, e.g., to inform the user that they will need to sign up for something in order to manage Semgrep across projects. I don’t have strong feelings though.